glean-core/ios/GleanTests/Metrics/PingTests.swift

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@testable import Glean
+import OHHTTPStubs
+import XCTest
+
+class PingTests: XCTestCase {
+    var expectation: XCTestExpectation?
+    var lastPingJson: [String: Any]?
+
+    override func setUp() {
+        lastPingJson = nil
+        expectation = nil
+
+        Glean.shared.resetGlean(clearStores: true)
+    }
+
+    private func setupHttpResponseStub(statusCode: Int32 = 200) {
+        stub(condition: isHost("incoming.telemetry.mozilla.org")) { data in
```

Comment:
I'm guilty of leaving this in my test code too, but it occurred to me when looking at this that if the telemetry endpoint changed in Glean, it would break this test.  Maybe relying on the constant in `Configuration` rather than a hardcoded endpoint would be safer.

Reply:
`Configuration.Constants.defaultTelemetryEndpoint` contains the `https://`, making this not easily work with `isHost`.

I'll take it to a follow-up